ProteoformSuiteInternal/ProteinSequenceGroup.cs

```diff
@@ -20,7 +21,7 @@ public ProteinSequenceGroup(IEnumerable<ProteinWithGoTerms> proteins_with_contam
                 proteins_with_contaminants_first.SelectMany(p => p.DatabaseReferences),
                 proteins_with_contaminants_first.SelectMany(p => p.GoTerms))
         {
-            List<ProteinWithGoTerms> proteinWithGoTermList = proteins_with_contaminants_first.ToList();
+            proteinWithGoTermList = proteins_with_contaminants_first.ToList();
```

Comment:
@acesnik Shouldn't this be on main branch too? I don't think anything is done with this list